library-best-practices/JLBP-6.md

```diff
@@ -115,8 +119,8 @@ Given this scenario, here are the possible combinations of renamings:
 Given the consequences, it seems clear that the two worst options are case 2
 (renaming the Maven ID while keeping the Java package the same) and case 3
 (renaming the Java package while keeping the Maven ID the same), and both should
-be avoided. The remaining three cases need to be weighed carefully. Among the
```

Comment:
@garrettjonesgoogle I felt this sentence was somewhat forgotten to be removed (in relation to the next sentence). Do you really need this sentence? If yes, I'll revert.

Reply:
Yeah this doesn't seem necessary to keep.

Reply:
Good. Thanks.